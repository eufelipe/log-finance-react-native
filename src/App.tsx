import React, {Fragment, Suspense} from 'react';

import Routes from 'routes';

import 'locales';
import {ActivityIndicator} from 'react-native';

const App = () => {
  return (
    <Fragment>
      <Suspense fallback={<ActivityIndicator />}>
        <Routes />
      </Suspense>
    </Fragment>
  );
};

export default App;
