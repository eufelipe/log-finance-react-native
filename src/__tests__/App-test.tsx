/* eslint-disable @typescript-eslint/unbound-method */
import React from 'react';
import {render} from '@testing-library/react-native';

import App from '../App';

describe('App', () => {
  it('Test match snapshot App', () => {
    // given
    const props = {};

    // when
    const {toJSON} = render(<App {...props} />);

    // then
    expect(toJSON()).toMatchSnapshot();
  });
});
