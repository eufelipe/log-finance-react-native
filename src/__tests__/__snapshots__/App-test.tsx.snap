// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test match snapshot App 1`] = `<ActivityIndicator />`;
